Text/Megaparsec/Error.hs

```diff
@@ -43,6 +43,7 @@ module Text.Megaparsec.Error
     errorBundlePretty,
     parseErrorPretty,
     parseErrorTextPretty,
+    showErrorItem,
```

Comment:
You should add `@since 9.4.0` to the documentation of this function (let's say the next release will be 9.4.0).